README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
I'm new to GitHub, come from a CVS, TFS, SVN background.  I can attempt to figure it out and do it later today or tomorrow.  I also would not be offended if someone else performed the tasks.